modules/file-extension-common/src/test/java/org/mule/runtime/module/extension/file/FilePredicateBuilderContractTestCase.java

```diff
@@ -269,6 +275,20 @@ public void rejectSymbolicLink()
         assertReject();
     }
 
+    @Test
+    public void failOnInvalidMinSize()
+    {
+        expectedException.expect(IllegalArgumentException.class);
```

Comment:
Why do you use a rule instead of using `@Test (expected=IllegalArgumentException.class)` if you are not validating the error message?